gradle.properties

```diff
@@ -21,7 +21,7 @@ publishPasswordProperty=ossrhPassword
 publishSignatureRequired=true
 
 # Gradle options
-org.gradle.jvmargs=-Xmx1792m -XX:+HeapDumpOnOutOfMemoryError
+org.gradle.jvmargs=-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError
```

Comment:
Since we run our CI with the parallel option, I thought we may need to increase the following value:
https://github.com/line/armeria/blob/713b89984a4c34866c42d32f8ef9fd72adbe4442/build.gradle#L52

(My understooding is that the `--parallel` option launches forked java processes, and one of these tasks is `:module:compile` which caused the oome)